esss_fix_format/cli.py

```diff
@@ -218,6 +218,10 @@ def get_files(cmd):
     result.update(get_files('git diff --name-only --diff-filter=ACM --staged'))
     result.update(get_files('git diff --name-only --diff-filter=ACM'))
     result.update(get_files('git ls-files -o --full-name --exclude-standard'))
+    # check_output returns bytes in Python 3
+    if sys.version_info[0] == 3:
+        result = [x.decode(sys.getfilesystemencoding()) for x in result]
```

Comment:
Hmm good point. There's no guarantee actually, after all programs are free to output anything in any encoding.

Since we are talking about `git` and asking for it to print out file-names, I think it is safe to assume it will output things using the file-system encoding.